packages/btcindexer/db/migrations/0001_initial_schema.sql

```diff
@@ -0,0 +1,24 @@
+-- This table tracks the blocks received from the relayer (queue for cron job)
+CREATE TABLE processed_blocks (
+    height INTEGER PRIMARY KEY,
```

Comment:
Block ID (hash) should be a primary key. We can have an index for height only if we use it in queries - let's add Todo to investigate later